Makefile

```diff
@@ -109,7 +109,7 @@ $(PROTOC_DOWNLOAD):
 
 $(PROTOC): $(PROTOC_DOWNLOAD)
 	@mkdir -p $(BIN)
-	unzip -DD -q $(PROTOC_DOWNLOAD) -d $(dir $(BIN)) bin/protoc
+	unzip -DD -o -q $(PROTOC_DOWNLOAD) -d $(dir $(BIN)) bin/protoc
```

Comment:
Had to add this flag for an update to work locally: `-o  overwrite files WITHOUT prompting`